lib/modulesync.rb

```diff
@@ -11,11 +11,19 @@
 require 'monkey_patches'
 
 GITHUB_TOKEN = ENV.fetch('GITHUB_TOKEN', '')
+GITHUB_ORGANIZATION = ENV.fetch('GITHUB_ORGANIZATION', '')
 
 Octokit.configure do |c|
   c.api_endpoint = ENV.fetch('GITHUB_BASE_URL', 'https://api.github.com')
+  c.auto_paginate = true
+
+  # We use a different Accept header by default here so we can get access
+  # to the developer's preview that enables repository topics.
+  c.default_media_type = ::Octokit::Preview::PREVIEW_TYPES[:topics]
 end
 
+GITHUB = Octokit::Client.new(:access_token => GITHUB_TOKEN)
```

Comment:
I'm not sure about a global with state. Perhaps it should be a lazy (static) method on the module. This can help in case it's used in an offline context.